pkg/daemon/config_drift_monitor_test.go

```diff
@@ -497,13 +532,34 @@ func (tc *configDriftMonitorTestCase) writeIgnitionConfig(t *testing.T, ignConfi
 	}
 
 	// Write systemd units the same way the MCD does.
-	if err := writeUnits(ignConfig.Systemd.Units, tc.systemdPath, true); err != nil {
+	if err := writeUnits(ignConfig.Systemd.Units, tc.rootPath, true); err != nil {
 		return fmt.Errorf("could not write systemd units: %w", err)
 	}
 
 	return nil
 }
 
+func createFalseSSHKeyDir(rootPath string, content []ign3types.SSHAuthorizedKey) error {
+	fpath := rootPath + "/home/core/.ssh/authorized_keys.d/ignition"
+	dir := filepath.Dir(fpath)
+	err := os.MkdirAll(dir, 0755)
+	if err != nil {
+		panic(err)
```

Comment:
question: Why not return the error back to the caller?

Reply:
agreed